amos-tub-ws18-proj1.iml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
How can this file be in your PR? It is stated in the `.gitignore` file of our project, please run `git rm -r --cached .`